run.js

```diff
@@ -103,13 +103,31 @@ const pkg = require('./package.json')
         state: 'open',
         labels: agendaLabel
       })
+
       console.log(`Fetching issues for ${r.owner}/${r.repo}: Found ${_agendaIssues.length}`)
       for (const i of _agendaIssues) {
         console.log(`Adding Issue: ${i.url}`)
         if (!agendaIssues.find((ii) => ii.url === i.url)) {
           agendaIssues.push(i)
         }
       }
+
+      const _agendaPr = await client.paginate('GET /repos/{owner}/{repo}/pulls', {
+        owner: r.owner,
+        repo: r.repo,
+        state: 'open',
+        labels: agendaLabel
+      })
+
+      for (const pr of _agendaPr) {
+        if (pr.labels.some(label => label.name === agendaLabel)) {
+          console.log(`Fetching PRs for ${r.owner}/${r.repo}: Found ${_agendaPr.length}`)
+          if (!agendaIssues.find((ii) => ii.url === pr.url)) {
```

Comment:
you are querying `agendaIssues` rather than `_agendaPr`  but more importantly, I think this workaround that was added to avoid duplicate issues is an artifact of the old GH SDK.  it should be safe to remove this check and just iterate both the issues and the PRs without needing this conditional